.gitignore

```diff
@@ -1,18 +1,19 @@
 /.idea
 /.composer/
-/vendor
+/vendor/
```

Comment:
👍 

What do you think about also sorting entries here

* by type (directories first, then files) first
* alphabetically second

?

Reply:
👍 I'll do that in a separate PR though